code/modules/mob/new_player/new_player.dm

```diff
@@ -420,6 +420,10 @@
 			if(is_species_whitelisted(chosen_species) || has_admin_rights())
 				new_character = new(loc, use_species_name)
 
+				if (istype(chosen_species, /datum/species/machine))

```

Comment:
At this point, would it be worth bringing it to `datum/species/machine/handle_post_spawn(var/mob/living/carbon/human/H)`?

```
datum/species/machine/handle_post_spawn(var/mob/living/carbon/human/H)
	H.gender = NEUTER
	. = ..()
	check_tag(H, H.client)
```